login_utils.py

```diff
@@ -136,13 +156,20 @@ def normal_login(session, username, password, continue_url):
         form_html = session.get(url_login)
 
     except(requests.exceptions.ConnectionError):
-        error = "Connection Error"
+        error = 504
         return None, error, session
 
     # payload to send with POST request, i.e. cookies, tokens etc. more details in
     # `utils.make_payload` function.
     payload = utils.make_payload(form_html.text)
 
+    # if the page did not have the form it won't have payload, that shows the response page has
+    # changed or the request was not appropriate.
+    if not payload:
+        file_name = utils.log_error("normal login", form_html.text)
+        error = 500
+        return form_html, error, session
+
     # add email, password and target url in payload
     payload['Email'] = username
```

Comment:
Asking just for curiosity you are taking some character some key e.g. Email & Passwd of payload dictionary in Capitalize way and  'continue' in lower case. Any reason?